clients/client-rust/download/src/test_helpers.rs

```diff
@@ -0,0 +1,139 @@
+//! Utilities for testing downloads
+use crate::service::{ObjectService, QueueService};
+use anyhow::Error;
+use async_trait::async_trait;
+use httptest::{matchers::*, responders::*, Expectation};
+use serde_json::Value;
+use std::fmt;
+use std::sync::{Arc, Mutex};
+
+const PLAINTEXT_BODY: &[u8] = b"hello, world";
+
+const GZIPPED_BODY: &[u8] = &[
```

Comment:
I always get disappointed when there isn't some fun string hidden in things like this!

Reply:
Sorry to disappoint!  I tried to compress it at runtime, but httptest's `.body(..)` wants a static string, so I just plunked it in a const.